components/VM_Arm/src/main.c

```diff
@@ -927,6 +941,28 @@ static int load_vm(vm_t *vm, const char *kernel_name, const char *dtb_name, cons
     return 0;
 }
 
+static unsigned long parse_camkes_vm_entry_addr(void)
+{
+    unsigned long addr = strtoul(vm_address_config.kernel_entry_addr, NULL, 0);
```

Comment:
Perhaps `uintptr_t` is more appropriate for these?

Reply:
Yes, this seems a better choice.

Reply:
Actually we have that type fix commit in our forks already, I can make a PR soon.

Reply:
Here: seL4/camkes-vm#58